spec/support/workflow_helpers.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module WorkflowHelpers
+  # resets the first errored out step in the given workflow
+  def reset_errored_workflow_step(workflow_name)
+    click_link workflow_name
+    select 'Rerun', from: 'status'
+    confirm_message = 'You have selected to manually change the status. '
+    confirm_message += 'This could result in processing errors. Are you sure you want to proceed?'
+    accept_confirm(confirm_message) do
+      click_button 'Save'
+    end
+  end
```

Comment:
Note: due to no good HTML selectors, it's not easy to properly select a specific step in the workflow for resetting... so this assumes there is a single step in error, and it will just reset that one (we can at least limit ourselves to a specific workflow)